pkg/gitqlite/display.go

```diff
@@ -37,6 +43,35 @@ func DisplayDB(rows *sql.Rows, w io.Writer, format string) error {
 	}
 	return nil
 }
+func single(rows *sql.Rows, write io.Writer) error {
+
+	columns, err := rows.Columns()
+	if err != nil {
+		return err
+	}
+	pointers := make([]interface{}, len(columns))
+	container := make([]sql.NullString, len(columns))
+
+	for i := range pointers {
+		pointers[i] = &container[i]
+	}
+	rows.Next()
+	err = rows.Scan(pointers...)
+	if err != nil {
+		return err
+	}
+
+	r := make([]string, len(columns))
+	for i, c := range container {
+		if c.Valid {
+			r[i] = c.String
+		}
+	}
+
+	fmt.Printf("%s : %s\n", columns[0], r[0])
```

Comment:
let's have this just print out the value and nothing else:

```go
fmt.Printf("%s\n", r[0])
```

as the main purpose of this format option is to allow easier piping of single value responses into other commands